src/printer.ts

```diff
@@ -230,6 +240,21 @@ export class PugPrinter {
 		return !!token && possibilities.includes(token.type) !== invert;
 	}
 
+	private getUnformattedContentLines(firstToken: Token, lastToken: Token): string[] {
+		const { start } = firstToken.loc;
+		const { end } = lastToken.loc;
+		const lines: string[] = this.content.split(/\r\n|\n|\r/);
+		const startLine: number = start.line - 1;
+		const endLine: number = end.line - 1;
+		const parts: string[] = [];
+		parts.push(lines[startLine].substring(start.column - 1));
```

Comment:
```suggestion
		parts.push(lines[startLine].slice(start.column - 1));
```

Reply:
`line` is a string, not an array?

Reply:
YES! I know: https://stackoverflow.com/questions/2243824/what-is-the-difference-between-string-slice-and-string-substring

Reply:
Crazy! All these years of writing JS and I never noticed that strings have a `slice()` method that does exactly the same as `substring()`. But why adding both? So strange 🤣